Core.ApplicationServices/Organizations/StsOrganizationSynchronizationService.cs

```diff
@@ -223,5 +261,44 @@ private Maybe<OperationError> Modify(Guid organizationUuid, Func<Organization, M
 
             return Maybe<OperationError>.None;
         }
+
+        private static Result<ExternalOrganizationUnit, OperationError> ConnectToExternalOrganizationHierarchy(
+            Organization organization, ExternalOrganizationUnit importRoot, Maybe<int> levelsToInclude, bool subscribeToUpdates)
+        {
+            return organization.ConnectToExternalOrganizationHierarchy(OrganizationUnitOrigin.STS_Organisation, importRoot, levelsToInclude, subscribeToUpdates)
+                .Match
+                (
+                    error => error,
+                    () => Result<ExternalOrganizationUnit, OperationError>.Success(importRoot));
+        }
+
+        private Result<StsOrganizationChangeLog, OperationError> WithLogEntries(OrganizationTreeUpdateConsequences consequences)
```

Comment:
this return value is now only used to produce the input... that makes no sense.. clean up and remove the added extension if this class is the only one using it